app/pharmacy/components/dispensation.js

```diff
@@ -76,13 +76,15 @@
       item.prescription = prescription;
       item.nextPickupDate = new Date();
       item.showNextPickupDate = true;
-      item.quantity = item.drugToPickUp;
+      item.quantity = getDefaultQuantityToDispense(item);
       updatePickupDate(item);
-
       vm.selectedPrescriptionItems = vm.selectedPrescriptionItems.concat([item]);
     }
 
-
+    function getDefaultQuantityToDispense(item){
```

Comment:
Shouldn't this business logic be in `dispensationService`?

Reply:
moved the business logic to dispensationService service. I don't think is good idea to put default quantity as proper of a resource, it is not attribute a but logic result